apollo-router/src/plugins/telemetry/config.rs

```diff
@@ -453,14 +452,30 @@ pub(crate) enum AttributeValue {
     Bool(bool),
     /// i64 values
     I64(i64),
+    /// i64 values
+    U128(u128),
     /// f64 values
     F64(f64),
     /// String values
     String(String),
+    // TODO add static string type support
```

Comment:
Not at all, that was because I thought we would have a lot of static attribute values but at the end not that much. So I think I'll delete that comment because for now we won't benefit from having that static support